examples/Cargo.toml

```diff
@@ -59,7 +59,8 @@ sp1-lib = { path = "../crates/zkvm/lib", default-features = false }
 sp1-zkvm = { path = "../crates/zkvm/entrypoint", default-features = false }
 
 [patch.crates-io]
-curve25519-dalek = { git = "https://github.com/sp1-patches/curve25519-dalek", branch = "patch-curve25519-v4.1.3" }
+ed25519-dalek = { git = "https://github.com/sp1-patches/ed25519-dalek.git", branch = "patch-ed25519-v1.0.1"}
+curve25519-dalek = { git = "https://github.com/sp1-patches/curve25519-dalek", branch = "patch-curve25519-v3.2.0"}
```

Comment:
Shouldn't you keep the original curve25519-dalek patch here? Or did that break your logic because it was patching the version of `curve25519-dalek` inside of `ed25519-dalek`?

Preferably, I'd like to have a patch for `ed25519-dalek` with `curve25519-dalek` version `3.x` and then a patch for `curve25519-dalek` with version `4.x`.

Reply:
It seems like I need to patch both ed25519 and curve25519-dalek version 3.x or I get errors